cmd/convert.go

```diff
@@ -93,7 +93,10 @@ func (o *convertOptions) validateArgs(args []string) error {
 
 func (o *convertOptions) run(plan string) error {
 	ctx := context.Background()
-	assets, err := o.readPlannedAssets(ctx, plan, o.project, o.ancestry, o.offline, false, o.rootOptions.errorLogger)
+	ancestryCache := map[string]string{
+		o.project: o.ancestry,
```

Comment:
It looks like o.project is expected to be a project id; I think that's probably fine for this PR? For the larger PR will this need to be prefixed with `projects/`?

Reply:
Yes, in theory, but I added in a parser in ancestrymanager to add the prefix if it does not exist.